src/workerd/api/http.c++

```diff
@@ -2067,8 +2067,16 @@ jsg::Promise<jsg::Ref<Response>> handleHttpResponse(jsg::Lock& js,
   if (isRedirectStatusCode(response.statusCode) &&
       jsRequest->getRedirectEnum() == Request::Redirect::FOLLOW) {
     KJ_IF_SOME(l, response.headers->get(kj::HttpHeaderId::LOCATION)) {
-      return handleHttpRedirectResponse(
-          js, kj::mv(fetcher), kj::mv(jsRequest), kj::mv(urlList), response.statusCode, l);
+
+      // Pump the response body to a singleton null stream before following the redirect.
```

Comment:
Just a nit (and probably not worth running CI over again just for this... so feel free to ignore) but it would probably be worthwhile expanding on this comment a bit more to explain why consuming the response body is necessary in this case... just to help future generations coming into this code later :-)

Reply:
Will do in follow up